### ISSUE TITLE
Problem in SSR finding

### ISSUE THREAD

**Issue by GauraviTrivedi** [lmdu/krait#19]
I am Gauravi Trivedi, I imported a sequence and GFF file in Krait. In order to find SSR, I clicked on "SSR" button without changing any parameters. Sequence got imported but the process of finding SSR is running without giving any output. If I click again on the SSR button, it pops up a text message saying "Task is still running". Can you please have a look at my issue and guide me through.
It will be really helpful.
Thank you

**Comment by lmdu** [comment]
Could you use version 1.3.3?. You can check the input file folder, is there a file with .fxi suffix generated? If no, it means your folder has no writable permission.

**Comment by GauraviTrivedi** [comment]
yes sir, ghe file with mentioned extension got generated by the latest
version of Krait 1.3.3

**Comment by GauraviTrivedi** [comment]
.fxi file is getting generated by Krait after importing a sequence. In order to find SSRs I clicked on SSR but no result table is shown by the software. In the task area, it is showing that "searching perfect SSRs for the imported sequence" but the result area is blank even after 30 minutes.
![image](https://user-images.githubusercontent.com/83008005/119294889-1e448500-bc73-11eb-9427-821dbe5ab445.png)

**Comment by lmdu** [comment]
It was so strange! I wonder if you could tell me your input sequence file. I can test it on my computer.
